tests/gold_tests/headers/via.gold

```diff
@@ -1,3 +1,4 @@
+Via: ---missing---
```

Comment:
Again, the observer begins recording before tests run, so it observes the health check happening before testing begins.